packages/node/src/index.ts

```diff
@@ -57,18 +57,11 @@ class YogaNodeServer<
     this.logger.debug('Setting up server.')
   }
 
-  getNodeServer(): NodeServer {
-    if (!this.nodeServer) {
-      this.endpoint = this.endpoint || '/graphql'
-      if (this.options?.https) {
-        this.nodeServer =
-          typeof this.options?.https === 'object'
-            ? createHttpsServer(this.options.https, this.requestListener)
-            : createHttpsServer(this.requestListener)
-      } else {
-        this.nodeServer = createHttpServer(this.requestListener)
-      }
-    }
+  /**
+   * @deprecated Will be removed in the next major. Get the server from `.start()` instead
+   * @returns The underlying Node.js server
+   */
+  getNodeServer(): NodeServer | null {
     return this.nodeServer
   }
```

Comment:
This would be a breaking change 🤔 

I think for now we still need to keep the get or create logic in here and simply deprecate it?